cgmes/cgmes-completion/src/main/java/com/powsybl/cgmes/completion/CreateMissingContainersPreProcessor.java

```diff
@@ -71,7 +74,7 @@ public class CreateMissingContainersPreProcessor implements CgmesImportPreProces
 
     public static final String NAME = "createMissingContainers";
     public static final String FIXES_FOLDER_NAME = "iidm.import.cgmes.fixes-for-missing-containers-folder";
-    public static final double DEFAULT_NOMINAL_VALUE_FOR_MISSING_VOLTAGE_LEVELS = 100.0;
+    public static final double DEFAULT_NOMINAL_VALUE_FOR_MISSING_VOLTAGE_LEVELS = 1.2345;
```

Comment:
What is the reason for this change?

Reply:
We do not want to use a "valid" (reasonable) nominal voltage that could be defined